CONTRIBUTING.md

```diff
@@ -44,36 +44,36 @@ dependency order.
 
 ## Build an individual package on the command line
 
-```
+```bash
 cd packages/<project>
 rushx build
 ```
 
 ## Run all tests for the whole repo
 
-```
+```bash
 rush test
 ```
 
 ## Start compile on save
 
 Starting this command will rebuild the typescript files on save.
 
-```
+```bash
 rush watch
 ```
 
 ## Cleanup
 
-Sometimes there is ghost files left in the dist folder, running this will get a clean state.
+Sometimes there are ghost files left in the dist folder(common when renaming a TypeScript file), running this will get a clean state.
```

Comment:
```suggestion
Sometimes there are ghost files left in the dist folder (common when renaming or deleting a TypeScript file), running this will get a clean state.
```